### ISSUE TITLE
Changes in imported files are not detected

### ISSUE THREAD

**Issue by bgstaal** [less/less.ruby#178]
Changes that are made in less files that are imported using the @import statement is not detected and therefore not rendered in browsers with localStorage enabled.

This makes it especially hard to work with several files imported into one during development.

example:

styles.less (included in html file via <link> tag)

<pre><code>@import "colors.less";</code></pre>

colors.less (exposed to html file via @import statement in styles.less)

<pre><code>body
{
    color: #FF0000;  // changes in this prop is not reflected in the rendered html after initial render
}</code></pre>

Workaround:

Disable browser localStorage

**Comment by Pym** [comment]
Please fix this, that's a very annoying issue...

**Comment by joeldrapper** [comment]
I agree this should be fixed, but for now you can always precompile your Less code into CSS using [Less.app](http://incident57.com/less/) if your on a Mac or [WinLess](http://winless.org) if your using Windows. Another good option is [SimpLess](http://wearekiss.com/simpless). This results in a much faster website anyway so is a better option really. Just make sure you hang on to the less file incase you want to make changes to it later.
